edgedb-protocol/src/value.rs

```diff
@@ -553,6 +579,24 @@ impl From<chrono::naive::NaiveTime> for LocalTime {
     }
 }
 
+#[cfg(test)]
+#[allow(dead_code)] // used by optional tests
+pub(self) mod test_helpers{
```

Comment:
We don't need an extra module here. `mod test` is generally is for helper functions. We could use bare `#[test]` functions as well if we don't need anything extra. But this is a minor issue.

Reply:
I extracted this module from a bigger restructuring in which there are multiple modules with their own test module each, several of which use these helpers. I'll probably prepare a first pull request for that this week.